eras/mary/impl/testlib/Test/Cardano/Ledger/Mary/ImpTest.hs

```diff
@@ -52,3 +56,15 @@ instance
   , Signable (DSIGN c) (Hash (HASH c) EraIndependentTxBody)
   ) =>
   MaryEraImp (MaryEra c)
+
+mkTokenMintingTx :: MaryEraImp era => ScriptHash (EraCrypto era) -> ImpTestM era (Tx era)
```

Comment:
We should mention this addition in the changelog and bump up the minor version, since that is a user facing feature.

Reply:
The Mary changelog already has a minor bump, from _Switch to using `ImpSpec` package_. However, I have added an extra entry for that version.